centreon/tests/e2e/package.json

```diff
@@ -24,6 +24,11 @@
     "filterSpecs": false,
     "nonGlobalStepDefinitions": true,
     "omitFiltered": true,
+    "json": {
+      "enabled": true,
+      "formatter": "./cypress/jsonFormatter/cucumber-json-formatter",
```

Comment:
Yes because we do not want to embed a binary file of 1.56MB in the git repository
@a-eljazouly is it possible to use the native json formatter ? https://github.com/badeball/cypress-cucumber-preprocessor/blob/master/docs/json-report.md

Reply:
@bdauria i don't think that it exists via npm.